docs/output.md

```diff
@@ -49,3 +50,37 @@ calm: string -> string
 pumped: string -> string
 edgy: string -> string
 ```
+
+## Line breaks / New lines / Blanks
+Since some `OutputPayloads` always take up the full width of the terminal (e.g., a `rule`), but others don't (e.g., `E "Hello"`) this lead us into making some opinionated choices for rendering the output. We want it to work as expected and with the least amount of manual _thinking about line breaks_ as possible.
```

Comment:
Clear explanation!
No need to change anything, but I want to declare that the term "manual thinking" made me chuckle :)

Reply:
Hmm, maybe it should be _cranial thinking_ instead :P